File: cpp/bc2702.cpp
#include <iostream>

using namespace std;

int main()
{
	int qRe[3], sPa[3], res = 0;
	
	cin >> qRe[0] >> qRe[1] >> qRe[2];
	cin >> sPa[0] >> sPa[1] >> sPa[2];
	
	for(int i = 0; i < 3; i++)
		if(qRe[i] < sPa[i]) res += sPa[i] - qRe[i];
	
	cout << res << endl;
}

File: python/bc2702.py
res = 0
qRe = [int] * 3
sPa = [int] * 3

fLi = input().split(" ")
sLi = input().split(" ")

for c in range(0, 3):
    qRe[c] = int(fLi[c])
    sPa[c] = int(sLi[c])

for i in range(0, 3):
    if qRe[i] < sPa[i]: res += sPa[i] - qRe[i]

print(res)
